### DISCUSSION TITLE
How to get featured in Showcase?

### DISCUSSION THREAD

**Discussion by pxeo** [prisma/prisma#16819] [Q&A] [answered]
We are building using Prisma and thought it would be a good idea to share with the community what we've been up to. [CoinRotator.app](coinrotator.app)

<img width="1273" alt="image" src="https://user-images.githubusercontent.com/100925074/207799749-85e8cf8c-f38c-45dc-a6f1-6d34fb0dd408.png">

**Comment by nurul3101** [comment, marked as answer]
Hello @pxeo 👋 

Can you please fill out this [Typeform](https://prisma103696.typeform.com/to/mArz7eo1)?
Our Team will get in touch with you with the next steps.

Thanks!

**Comment by ludralph** [comment]
Hi there,

To keep our discussions organized and focused on the most relevant topics, we’re reviewing and tidying up our backlog. As part of this process, we’re closing discussions that have already been marked as answered but remain open.

If this discussion still requires further input or clarification, feel free to reopen it or start a new one with updated details. Your contributions are invaluable to the community, and we’re here to help!

For more details about our priorities and vision for the future of Prisma ORM, check out our latest blog post: https://www.prisma.io/blog/prisma-orm-manifesto.

Thank you for your understanding and ongoing support of the Prisma community!
